frontend/src/App.js

```diff
@@ -1,11 +1,38 @@
+import { useState, useEffect } from 'react'
+import getProducts from './services/getProducts';
+import getProductById from './services/getProductById';
 
+export default function App() {
+
+  const [products, setProducts] = useState([])
+  const [productById, setProductById] = useState([])
+
+  useEffect(() => {
+    getProducts()
+    .then(data => setProducts(data))
+    .catch(error => console.log(error))
+  }, [])
+
+  useEffect(() => {
+    const productId = 2
+    getProductById(productId)
+    .then(data => setProductById(data))
+    .catch(error => console.log(error))
+  }, [])
```

Comment:
Why are you fetching the product with id 2 static?

Reply:
I just wanted to test the functionality of the fetch. Later a list of products will be fetched and displayed in a gallery so that the user can compare the products and the amount of virtual water which is used for the production.